recipes/pylatexenc/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pylatexenc" %}
+{% set version = "2.1" %}
+{% set sha256 = "bde5ffe8ee0c159da2845f1489401ee4246a844d06b452236e59880bd6c2d6ee" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/phfaist/pylatexenc/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
```

Comment:
I see there are entry points in setup.py:
```
    entry_points = {
        'console_scripts': [
            'latexwalker=pylatexenc.latexwalker.__main__:main',
            'latex2text=pylatexenc.latex2text.__main__:main',
            'latexencode=pylatexenc.latexencode.__main__:main',
```
Do you know if this package can be used from the command line, e.g. `latexwalker  -h`?

If so, that please add the entry points here, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points, and test for them as commands in the test section.

Reply:
Looks like it does. Will do.